qatrack/service_log/admin.py

```
@@ -142,6 +142,11 @@ def delete_view(self, request, object_id, extra_context=None):
 
         return super().delete_view(request, object_id, extra_context)
 
+    @admin.display(description=_l("Color"))
+    @mark_safe
```

Ruff (0.15.15)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)